saleor/account/emails.py

```diff
@@ -23,12 +27,22 @@ def send_password_reset_email(context, recipient):
 
 @shared_task
 def send_account_delete_confirmation_email(token, recipient_email):
-    site = Site.objects.get_current()
     delete_url = build_absolute_uri(
         reverse('account:delete-confirm', kwargs={'token': token}))
-    ctx = {'site_name': site.name, 'domain': site.domain, 'url': delete_url}
+    ctx = get_email_base_context()
+    ctx.update({'url': delete_url})
```

Comment:
I would recommend not calling `update(...)` (everywhere you put it) as there is no use to do that, it costs more CPU time for nothing in the current context.

Just do `ctx['delete_url'] = delete_url`